cloud-sdk-tool-build-task/task.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "$schema": "https://raw.githubusercontent.com/AArnott/vsts-task-lib/d8a200950849839678f6e34d241f4418ed5d2678/tasks.schema.json",
```

Comment:
Just curious, is there a reason why we use this particular link instead of the aka.ms one?

Reply:
There used to not be an aka.ms one. It finally got merged in [Microsoft/vsts-task-lib#146](https://github.com/Microsoft/vsts-task-lib/pull/146)